spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -65,6 +76,16 @@ Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, no
  - https://jakarta.ee/specifications/restful-ws/2.1/restful-ws-spec-2.1.html
  - https://github.com/eclipse-ee4j/jaxrs-api
 
+==== Optional Requirements
+
+MicroProfile 4.x implementations may opt to not implement the following JAX-RS features:
```

Comment:
```suggestion
The following JAX-RS features have been made optional in the MicroProfile 4.x implementations.
```

Reply:
Same comment here...  MicroProfile can not declare portions of the JAX-RS Spec as being optional.  Only the JAX-RS team can declare those type of things.

Reply:
It might be a bit sticky on JAX-RS when moving forward. In the current Jakarta EE 10, there is no plan to split JAX-RS to address the requirement for MP. Even though Jakarta Core Profile 1.0 is released, MP might still not be able to consume it. @starksm64 do we need a JAX-RS lite to be included in Jakarta Core Profile?